### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by OzanYRT**
Finished constant handling and did more checkstyle, remaining checkstyle are mostly javaDocs.

### CODE REVIEW

**Review by bli129 — APPROVED** [review]
looks good to mee

File: src/main/java/org/bcit/comp2522/project/ConstantManager.java [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
   public static final int ONE = 1;
   public static final int TWO = 2;
-  public static final int TWOHUNDRED = 200;
-  public static final int TWOTWENTY = 220;
-  public static final int TWENTYFOUR = 24;
-  public static final float TWOF = 2f;
-  public static final float POINTNINETYFIVE = 0.95f;
-  public static final float POINTNINE = 0.9f;
-  public static final float POINTFIFTEEN = 0.15f;
+  public static final int ENEMY_SPAWNER_UI = 200;
```
Comment (bli129) [inline src/main/java/org/bcit/comp2522/project/ConstantManager.java:12]: very cool

**Review by breader1 — CHANGES_REQUESTED** [review]
@author tags are class level please.

There was a private variable changed to public, please change back.

**Review by breader1 — APPROVED** [review]
cool thanks :)

**Pull request merged**